runtime/arc.js

```diff
@@ -205,14 +207,16 @@ class Arc {
       // TODO: pass slot-connections instead
       this.pec.slotComposer.initializeRecipe(particles);
     }
+
+    let newRecipe = new Recipe();
+    newRecipe.particles = particles;
```

Comment:
This doesn't seem quite right, the particles/views/slots setters don't seem to wire things up correctly, but I guess those details don't matter for your purposes.

Reply:
I think it does the right thing (despite how it looks, and it looks bad, i agree))) -it just assigns the lists of particles, views and slots that are already wired up correctly. 

I didn't manage to copy from |recipe| to both the _activeRecipe and the list of recipes and to end up with the same objects in both. 

This seems to work fine with the demo now, so i'll merge as is and put it in the agenda to discuss in more detail next week. Thanks